bin/i18n/hoc_sync_utils.rb

```diff
@@ -63,6 +63,11 @@ def self.rename_downloads_from_crowdin_code_to_locale
       File.write(new_path, new_translation_data.to_yaml)
       FileUtils.rm old_path
     end
+
+    # Now, any remaining directories named after the language name (rather than
+    # the four-letter language code) represent languages downloaded from
+    # crowdin that aren't in our system. Remove them.
+    FileUtils.rm_r(Dir.glob(File.join(I18N_SOURCE_DIR, "hourofcode", "[A-Z]*")))
```

Comment:
Is this normal? Is this a bad state we would want to log a warning about so someone knows to go clean it up / investigate?

Reply:
It's normal, and not a bad state; we expect it to happen whenever a language has been enabled for our project on crowdin before it gets added to the list of officially supported languages. We do this pretty often because that allows us to start collecting translations for a language in advance of enabling it.

I'll add an explanatory comment!